content/engine/install/rhel.md

```diff
@@ -16,15 +16,12 @@ aliases:
 download-url-base: https://download.docker.com/linux/rhel
 ---
 
-> **Note**
+> **Experimental**
 >
-> The installation instructions on this page refer to packages for RHEL on the
-> **s390x** architecture (IBM Z). Other architectures, including x86_64, aren't
-> yet supported for RHEL.
->
-> For other architectures, you may be able to install the CentOS packages.
-> Refer to [Install Docker Engine on CentOS](centos.md).
-{ .warning }
+> Support for x86_64 and aarch64 on RHEL is experimental.
+> If you have feedback about RHEL on these architectures, please reach out through our
```

Comment:
I'm a bit confused though; the _docker desktop_ packages are experimental / early access, but should that also be put here? The engine itself is not even used from this package (only the CLI packages are used by DD4L); I wonder if this makes it more confusing how DD !== Docker Engine by associating them here.

Possibly it would make more sense to have a banner "Docker Desktop for Linux is now available for RHEL" (that we can put at the top of relevant pages)

Reply:
DD for RHEL is not available yet except for early access program, so we are pointing users there.

Some users are just seeing the packages on download.docker.com and asking about it.

Reply:
Changed the wording...